File: app/src/main/java/com/kruma/kanta/service/IUbicacion.java
package com.kruma.kanta.service;

import com.kruma.core.service.entity.ServiceCallback;
import com.kruma.kanta.entidad.Ubicacion;


import retrofit.http.Body;
import retrofit.http.GET;
import retrofit.http.POST;
import retrofit.http.Path;

/**
 * Created by JORDY on 20/03/2017.
 */

public interface IUbicacion {

    @GET("/Obtener/{IdLocal}/{IdUbicacion}")
    void Obtener(
            @Path("IdLocal") String str_pIdLocal,
            @Path("IdUbicacion")String str_pIdUbicacion,
            ServiceCallback<Ubicacion> obj_pCallBack);

    @POST("/ObtenerQR")
    void ObtenerQR(
            @Body String QR,
            ServiceCallback<Ubicacion> obj_pCallBack);

}

File: KantaPe_Web/Kruma.KantaPe/Kruma.KantaPe.Service/App_Code/Interface/IUbicacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.ServiceModel.Web;
using System.Text;

namespace Kruma.KantaPe.Service.Interface
{
    [ServiceContract]
    public interface IUbicacion
    {
        [OperationContract]
        [WebInvoke(
        Method = "GET",
        ResponseFormat = WebMessageFormat.Json,
        BodyStyle = WebMessageBodyStyle.Bare,
        UriTemplate = "Obtener/{IdLocal}/{IdUbicacion}")]
        Kruma.KantaPe.Service.Entity.ServiceResult<Kruma.KantaPe.Entidad.Ubicacion> Obtener(string IdLocal, string IdUbicacion);

        [OperationContract]
        [WebInvoke(
        Method = "POST",
        ResponseFormat = WebMessageFormat.Json,
        BodyStyle = WebMessageBodyStyle.Bare,
        UriTemplate = "ObtenerQR")]
            Kruma.KantaPe.Service.Entity.ServiceResult<Kruma.KantaPe.Entidad.Ubicacion> ObtenerQR(string QR);
    }
}
